packages/storybook/stories/MessageThread/MessageThread.stories.tsx

```diff
@@ -428,6 +439,23 @@ const Docs: () => JSX.Element = () => {
         </Canvas>
       </div>
 
+      <div ref={refRichTextEditorInlineImages}>
+        <Heading>Rich Text Editor Support for Editing Messages with Inline Images</Heading>
+        <SingleLineBetaBanner />
+        <Description>
+          The following examples show how to enable image insert functionality for message editing with rich text
```

Comment:
Should we also include information about onRemoveInlineImage callback?

Reply:
> Should we also include information about onRemoveInlineImage (onCancelInlineImageUpload) callback?

I will add that after the API changes because the callback might be different.